emcc.py

```diff
@@ -395,6 +395,34 @@ def apply_settings(user_settings):
     if key == 'WASM_OBJECT_FILES':
       settings.LTO = 0 if value else 'full'
 
+  wasm_bigint_min_browser_versions(user_settings)
+
+
+def wasm_bigint_min_browser_versions(user_settings):
+  if not settings.WASM_BIGINT:
+    return
+
+  # See https://caniuse.com/?search=bigint
+  min_bigint_versions = {"Chrome": 67, "Edge": 79, "Firefox": 68, }
+
+  for browser, version in min_bigint_versions.items():
```

Comment:
I think perhaps we should not error here. Instead, let's just change the default, but still leave it possible for the user to override it. Perhaps the user has some special reason for doing so.

This will also make the code simpler, as then these could be
```python
default_setting(user_settings, 'MIN_CHROME', ..)
```

Reply:
Ok.